crates/miden-lib/asm/kernels/transaction/lib/account.masm

```diff
@@ -314,38 +285,60 @@ export.get_initial_commitment
         exec.memory::get_init_account_commitment
     else
         # for the foreign account current commitment equals to initial
-        exec.compute_current_commitment
+        exec.compute_commitment
     end
     # => [INIT_COMMITMENT]
 end
 
-#! Returns the vault root of the active account at the beginning of the transaction.
+#! Computes the commitment of the active account.
```

Comment:
nit: maybe I'd write this as "Computes commitment to the state of the active account."